app/views/ctc/questions/spouse_drivers_license/edit.html.erb

```diff
@@ -0,0 +1,33 @@
+<% content_for :page_title, t("views.ctc.questions.spouse_drivers_license.title") %>
```

Comment:
i do actually think that it makes sense to maybe use a partial for this, but its not a sticking point.